README.md

```diff
@@ -52,16 +52,27 @@ The assigment was to create a full functioning to-do app where the user can:
 
 Code review written by [Nelly Svarvare Petrén](https://github.com/NellySP).
 
-1. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-2. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-3. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-4. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-5. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-6. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-7. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-8. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-9. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
-10. `example.js:10-15` - Remember to think about X and this could be refactored using the amazing Y function.
+1. signup.php 17, 23, 29, 35. Below the inlog and signup-form it says:  please provide the your password/email/name. An extremely minor thing!
+
+2. It seems you can create tasks without adding it to a list. Is this intentional? Otherwise this could be fixed with a -> elseif (!isset($_POST[‘list-title'])) { $_SESSION['message'] = 'You have to assign your task to a list!’;
```

Comment:
Yes, this is a feature :) If you just want one list you can have all your tasks under "All tasks".